man/cppcheck.1.xml

```diff
@@ -135,6 +135,7 @@ man(1), man(7), http://www.tldp.org/HOWTO/Man-Page/
       <arg choice="opt"><option>--std=&lt;id&gt;</option></arg>
       <arg choice="opt"><option>--suppress=&lt;spec&gt;</option></arg>
       <arg choice="opt"><option>--suppressions-list=&lt;file&gt;</option></arg>
+	  <arg choice="opt"><option>--suppress-xml=&lt;.xml file&gt;</option></arg>
```

Comment:
the indentation looks suspicious. as far as I see the previous line uses spaces and your line uses a tab.